why3/src/exp.rs

```diff
@@ -50,11 +54,15 @@ impl BinOp {
             BinOp::Ne => Infix1,
             BinOp::Ge => Infix1,
             BinOp::Gt => Infix1,
-            BinOp::FloatAdd => Infix4,
-            BinOp::FloatSub => Infix4,
-            BinOp::FloatMul => Infix4,
-            BinOp::FloatDiv => Infix4,
+            BinOp::FloatAdd => Infix5,
```

Comment:
I added a new precedence level to separate arithmetic from comparison for float operators. That felt sensible to me.

Reply:
Hmm the why manual lists the precedence levels of operators and how they should spread out throughout them. According to that all float operators should be Infix4